crates/typescript-tests/run.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+cd `dirname "$0"`
```

Comment:
This script heavily relies on relative path. This makes sure that the script can be run anywhere without breaking it.